Formula/android-sdk.rb

```diff
@@ -30,11 +30,11 @@ class AndroidSdk < Formula
 
   # Version of the android-build-tools the wrapper scripts reference.
   def build_tools_version
-    "23.0.1"
+    "25.0.2"
   end
 
   def install
-    prefix.install "tools", "SDK Readme.txt" => "README"
+    (prefix/"tools").install
```

Comment:
Sorry, I just meant that would replace `prefix.install`. You still need to specify what is installed, i.e. `Dir["*"]`